src/components/investigate-dialog.sass

```diff
@@ -0,0 +1,99 @@
+@import vars
+
+.investigate-dialog
+  display: grid
+  grid-template-rows: $investigate-dialog-header-height auto $investigate-dialog-footer-height
+  grid-template-columns: $investigate-dialog-width
+  grid-gap: $component-margin
+  justify-content: center
+  text-align: center
+  background-color: $component-bgcolor
+  width: $investigate-dialog-width
+  height: $investigate-dialog-height
+  border: 3px solid $component-bgcolor
+  border-radius: $dialog-border-radius
+  opacity: 1
+  -webkit-animation-name: fadein /* Safari 4.0 - 8.0 */
+  -webkit-animation-duration: .5s /* Safari 4.0 - 8.0 */
+  animation-name: fadein
+  animation-duration: .5s
+
+/* Safari 4.0 - 8.0 */
+@-webkit-keyframes fadein
```

Comment:
`autoprefixer` should make these prefixes unnecessary, though I haven't tested it myself.

Reply:
Just tested.  From the generated CSS, it looks like it works!